repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -72,6 +72,12 @@ export interface Direction {
     inboundDirection?: string;
 }
 
+export interface EditFareStageMatchingWithErrors {
+    warning?: boolean;
```

Comment:
we should avoid optional keys as much as we can. warning looks like it can be 'false' when we dont want to use it, and errors can be an empty array.

Reply:
@NitishAroraKPMG  still not resolved